release_notes.md

```diff
@@ -1,3 +1,6 @@
+## NEXT
```

Comment:
If you have modified the version in the package.json, then you should have the same version in your release notes